proof/invariant-abstract/InvariantsPre_AI.thy

```diff
@@ -8,15 +8,12 @@ theory InvariantsPre_AI
 imports LevityCatch_AI
 begin
 
-context begin interpretation Arch .
-
-requalify_types
-  aa_type
-
-requalify_consts
+(* FIXME RAF: these already appear exported to global context?!
+arch_requalify_types (A)
   aa_type
```

Comment:
Entirely possible that we have a bunch of duplication for some of these, because we didn't have warning before and would not have noticed without explicitly checking for it, e.g. after things got moved around at some point.

Reply:
These were already exported in ASpec it turns out.